predicates/traffic/traffic.go

```diff
@@ -0,0 +1,140 @@
+/*
+Package traffic implements a custom predicate to control
+traffic by setting rates for certain routes.
```

Comment:
"Package traffic implements a predicate to control the matching probability for a given route by setting its weight"